src/components/chat/tool-icon-component.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { useContext } from "react";
+import { useTypeOfTileInDocumentOrCurriculum } from "../../hooks/use-stores";
+import { AppConfigContext } from "../../app-config-context";
+
+interface IProps {
+  documentKey?: string;
+  tileId?: string;
+}
+
+export const ToolIconComponent: React.FC<IProps> = ({documentKey, tileId}) => {
+
+  const tileType = useTypeOfTileInDocumentOrCurriculum(documentKey, tileId);
+  const { appIcons } = useContext(AppConfigContext);
+  
+  if (!documentKey || !tileId || !appIcons ) {
```

Comment:
There is an extra space here too after `!appIcons`:
`if (!documentKey || !tileId || !appIcons) {`

Reply:
Changed.